message_ix_buildings/chilled/util/config.py

```diff
@@ -0,0 +1,194 @@
+import datetime
+from dataclasses import dataclass
+from typing import Literal
+
+import numpy as np
+
+from message_ix_buildings.chilled.functions.user_settings import DICT_USER_SETTINGS
+
+
+@dataclass
+class Config:
+    """Configuration for :mod:`.message_ix_buildings.chilled`."""
+
+    #: Select user to set the paths to the project and data directories.
+    #:
+    #: One of: "ALE", "ED", "MEAS", "MEAS_EBRO", "MEAS_UNICC".
+    user: Literal["ALE", "ED", "MEAS", "MEAS_EBRO", "MEAS_UNICC"] = "MEAS_UNICC"
+
+    #: Set the version of the module run.
+    #:
+    #: This is used to name the output files and directories.
+    vstr: str = "ALPS2023"
+
+    #: Select the climate model.
+    #:
+    #: One of: "GFDL-ESM4", "IPSL-CM6A-LR", "MPI-ESM1-2-HR", "MRI-ESM2-0", "UKESM1-0-LL"
+    gcm: Literal[
+        "GFDL-ESM4", "IPSL-CM6A-LR", "MPI-ESM1-2-HR", "MRI-ESM2-0", "UKESM1-0-LL"
+    ] = "GFDL-ESM4"
+
+    #: Select RCP scenario.
+    #:
+    #: One of: "ssp126", "ssp370", "ssp585", "baseline".
+    rcp: Literal["ssp126", "ssp370", "ssp585", "baseline"] = "baseline"
+
+    #: List all possible RCP scenarios.
+    rcps = ["ssp126", "ssp370", "ssp585", "baseline"]
+
+    #: Set what data to use for the RCP scenario.
+    #: If rcp is "baseline", then use "ssp126" data.
+    #: Otherwise, use the data corresponding to the RCP scenario selected.
+    if rcp == "baseline":
+        rcpdata = "ssp126"
+    else:
+        rcpdata = rcp
```

Comment:
```suggestion
    rcpdata = "ssp126" if rcp == "baseline" else rcp
```
Something like this should also work and is generally regarded "less complex".

Reply:
(Might be that the line is then too long for ruff's liking or so.)